book/generated-snippets/ts/index.snippet.message-ext-submit.ts

```diff
@@ -0,0 +1,21 @@
+app.on('message.ext.submit', async ({ send, activity }) => {
+  const { commandId } = activity.value;
+  let card: Card;
+
+  if (commandId === 'createCard') {
+    // activity.value.commandContext == "compose"
```

Comment:
Do we want to keep this comment?

Reply:
i want to convey that both of these surfaces trigger the same activity

Reply:
Also that it's the same as what's defined in the manifest: 

![image](https://github.com/user-attachments/assets/66c35835-9474-4bd5-b266-ad83735840da)